<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>multiTripImportDialog</class>
 <widget class="QDialog" name="multiTripImportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>163</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import New Trip</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>File</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>110</y>
     <width>97</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="importButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>110</y>
     <width>97</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
  </widget>
  <widget class="QPushButton" name="browseButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>40</y>
     <width>97</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse..</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="locationInputField">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>43</y>
     <width>371</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
